deployment/tokens/internal/seqs/seq_persist_address.go

```diff
@@ -0,0 +1,74 @@
+package seqs
+
+import (
+	"github.com/Masterminds/semver/v3"
+
+	"github.com/smartcontractkit/chainlink-deployments-framework/datastore"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+	"github.com/smartcontractkit/chainlink/deployment/tokens/internal/ops"
+)
+
+// SeqPersistAddressDeps contains the dependencies for the SeqPersistAddress sequence.
+type SeqPersistAddressDeps struct {
+	AddrBook  cldf.AddressBook
+	Datastore datastore.MutableDataStore[
+		datastore.DefaultMetadata,
+		datastore.DefaultMetadata,
+	]
+}
+
+// SeqPersistAddressInput is the input to the SeqPersistAddress sequence.
+type SeqPersistAddressInput struct {
```

Comment:
looks like it should be a generic sequence not specific to tokens. makes sense to move it out to common?

Reply:
I've left this in `internal` for now while we explore how we want to expose common operations through the framework.